etcd/assets/configuration/spec.yaml

```diff
@@ -8,7 +8,9 @@ files:
   - template: instances
     options:
       - name: use_preview
```

Comment:
I'm not sure if `use_preview` is still the best name for this config, maybe `use_prometheus` may be better? We could update this config name once OpenMetrics is added for this check.